tedana/decomposition/eigendecomp.py

```diff
@@ -216,14 +214,36 @@ def tedica(n_components, dd, conv, fixed_seed, cost, final_cost):
     Uses `mdp` implementation of FastICA for decomposition
     """
 
-    import mdp
+    from sklearn.decomposition import FastICA
+
+    def cost_func(cost):
+        """
+        Cost function supplied to ICA.
+
+        Parameters
+        ----------
+        cost:  {'tanh', 'pow3', 'gaus', 'skew'} str
+        """
+        def gaus(x):
+            return x * np.exp(-1 * x**2 / 2)
+
+        def skew(x):
+            return x**2
+
+        if cost is 'tanh':
+            return 'logcosh'
+        if cost is 'pow3':
+            return 'cube'
+        if cost is 'gaus':
+            return gaus
+        if cost is 'skew':
+            return skew
```

Comment:
Why not change `tanh` to `logcosh` and `pow3` to `cube` here? It's a breaking change anyway, right?
Also, can you add an exception if `cost` is none of those?